enderio-base/src/main/java/com/enderio/base/common/capability/darksteel/DarkSteelUpgradeable.java

```diff
@@ -0,0 +1,161 @@
+package com.enderio.base.common.capability.darksteel;
+
+import com.enderio.base.EnderIO;
+import com.enderio.base.common.capability.EIOCapabilities;
+import com.enderio.base.common.item.darksteel.upgrades.DarkSteelUpgradeRegistry;
+import com.enderio.base.common.item.darksteel.upgrades.EmpoweredUpgrade;
+import net.minecraft.nbt.CompoundTag;
+import net.minecraft.nbt.Tag;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.world.item.ItemStack;
+
+import java.util.*;
+
+@SuppressWarnings("unused")
+public class DarkSteelUpgradeable implements IDarkSteelUpgradable {
+
+    //----------------------- Utils
```

Comment:
Could we possibly swap to #region #endregion like in some of my other commits? I personally think it looks cleaner. (Note this is in many classes). This is just my opinion however :D

Reply:
Done in all the places